ddtrace/internal/writer.py

```diff
@@ -191,13 +208,18 @@ def run_periodic(self):
             self.dogstatsd.gauge("datadog.tracer.heartbeat", 1)
 
         try:
-            self.flush_queue()
+            dropped_root_spans = self.flush_queue()
+        except Exception:
+            dropped_root_spans = 0
         finally:
+            dropped, enqueued, enqueued_lengths = self._trace_queue.pop_stats()
+
+            self._drop_sma.set(float(dropped_root_spans + dropped) / float(enqueued))
```

Comment:
```suggestion
            self._drop_sma.set(float(dropped_root_spans + dropped) / enqueued)
```
only one of the types needs to be a float for the result to be a float (for python 2)